### ISSUE TITLE
Open a new Window with a PDF

### ISSUE THREAD

**Issue by marcgauch** [nklayman/vue-cli-plugin-electron-builder#1129]
**Is your feature request related to a problem? Please describe.**
I am wondering how to open a new window and display a PDF.

**Describe the solution you'd like**
I'd like a brief instruction on how to do it. How can I call a function of electron within the context of a Vue file. I found [this](https://nklayman.github.io/vue-cli-plugin-electron-builder/guide/recipes.html#multiple-pages). But I'd rather open it once the user needs it.
tl;dr: Click on a button and open a new window with a pdf.

**Describe alternatives you've considered**
none

**Additional context**
I found [this issue](https://github.com/nklayman/vue-cli-plugin-electron-builder/issues/963) but couldn't find the issue that they should have created.

**Comment by nklayman** [comment]
All you should need to do is create a link to the pdf file with a target of `_blank`:
```html
<a href="http://site.com/somePDF.pdf" target="_blank">Open pdf</a>
```
